tests/test_rest_api.py

```diff
@@ -40,10 +40,11 @@ def test_liveness_endpoint(client):
 
 def test_register_api_endpoint(client):
     """Test function for register endpoint."""
-    reg_resp = client.post(api_route_for("register"), data=json.dumps(payload))
+    reg_resp = client.post(api_route_for("register"),
+                           data=json.dumps(payload), content_type='application/json')
     assert reg_resp.status_code == 200
     jsn = get_json_from_response(reg_resp)
-    assert(jsn["success"] == true)
+    assert(jsn["success"] is True)
```

Comment:
or just `assert jsn["success"]`

Reply:
Sure @jpopelka 

These tests are failing because following env variables are required
PGBOUNCER_SERVICE_PORT
 POSTGRESQL_USER
 POSTGRESQL_PASSWORD
 POSTGRESQL_DATABASE
PGBOUNCER_SERVICE_HOST

@jpopelka and @tisnik, is there a way by which we can verify if these variables are present or not?
If I export them in my terminal and run the tests, they work perfectly fine. I really need help here.